test/merkle-patricia-proof.js

```diff
@@ -77,12 +77,12 @@ describe('Check proof of Merkle Patricia tree', function() {
 
     it('should throw error when keys one key is full prefix of another', function() {
         var data = require('./data/invalid-merkle-patricia-tree-with-common-prefix-children.json');
-        expect(() => Exonum.merklePatriciaProof(data.root_hash, data.proof, data.searched_key)).to.throw(Error);
+        expect(() => Exonum.merklePatriciaProof(data.root_hash, data.proof, data.searched_key)).to.throw(Error, 'Impossible to determine left and right nodes.');
```

Comment:
Pro-tip: you can just do `.throw('Impossible to determine left and right nodes.')` assuming you're smart enough not to throw raw strings or something else not inheriting from `Error`. You can also use regular expressions: `.throw(/left.*right/i)`. Also, could you please split this line?